src/shared-components/dropdown/index.tsx

```diff
@@ -5,6 +5,20 @@ import MobileDropdown from './mobileDropdown';
 import DesktopDropdown from './desktopDropdown';
 import allowNullPropType from '../../utils/allowNullPropType';
 
+export type OptionType = {
```

Comment:
This file is just TS conversion changes.

Reply:
@ipc103 I think we'll also want to convert this to a Function component. We're typing `optionsContainerMaxHeight` as optional in children components, but it's provided via `defaultProps`. If we convert this to a Function component we can rely on ES6 defaults to inform TypeScript that it is optional but, downstream, being required is not type inconsistent.